### ISSUE TITLE
UndefVarError: Error not defined

### ISSUE THREAD

**Issue by oliviaAB** [alanderos91/BioSimulator.jl#8]
Hi,

When I'm running simulations sometimes I get the following message:
```{julia}
UndefVarError: Error not defined
```
I think this is because the error messages are displayed using for example
```{julia}
 throw(Error("intensity = $(intensity(a)) < 0 at time $algorithm.t"))
```
instead of 
```{julia}
throw(error(...))
```
Is it a change necessary for Julia v0.7? (I'm still using v0.6.2)

Thank you!

**Comment by alanderos91** [comment]
This was accidentally introduced in an old commit. Should be fixed in the latest release once it is merged.

**Comment by alanderos91** [comment]
See the v0.4 release (08f118180969431b0a4ef43ec10d970e343b22e9). FYI I have a Julia v0.7 upgrade in place, it just needs to be merged into master.

**Comment by oliviaAB** [comment]
Thank you!
